### PR TITLE
Throws exception on use integer key in JS/CSS options + Increase test coverage

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by devanych — APPROVED** [review]

File: src/AssetManager.php [hunk 1]
```diff
@@ -834,16 +834,22 @@ private function isAllowedBundleDependencies(string $name, AssetBundle $bundle):
...
+    private function mergeOptionsWithArray(array $options, array $array): array
     {
-        foreach ($a as $key => $value) {
+        foreach ($options as $key => $value) {
             if (is_int($key)) {
-                $b[] = $value;
-            } elseif (!array_key_exists($key, $b)) {
-                $b[$key] = $value;
+                throw new InvalidConfigException(
+                    'JavaScript or CSS options should be list of key/value pairs with string ketys. Got integer key.'
```
Comment (mamadza) [inline src/AssetManager.php:845]: 'JavaScript or CSS options should be list of key/value pairs with string keys. Got integer key.'

File: tests/AssetManagerTest.php [hunk 2]
```diff
@@ -577,6 +618,98 @@ public function testAppendTimestamp(): void
...
+        $manager->register([JsVarAsArrayWithIntegerNameAsset::class]);
+    }
+
+    public function testJsOptionsWithIntegerKey(): void
+    {
+        $manager = $this->createManager();
+
+        $this->expectException(InvalidConfigException::class);
+        $this->expectExceptionMessage(
+            'JavaScript or CSS options should be list of key/value pairs with string ketys. Got integer key.'
```
Comment (mamadza) [inline tests/AssetManagerTest.php:699]: 'JavaScript or CSS options should be list of key/value pairs with string keys. Got integer key.'

File: src/AssetManager.php (hunk 1 above)
Comment (vjik) [reply src/AssetManager.php:845]: what is the question?

File: tests/AssetManagerTest.php (hunk 2 above)
Comment (vjik) [reply tests/AssetManagerTest.php:699]: what is the question?

File: src/AssetManager.php (hunk 1 above)
Comment (mamadza) [reply src/AssetManager.php:845]: just correct keys 
It's my first time

Comment (vjik) [reply src/AssetManager.php:845]: Thank you. 
I didn't notice it at first.

**Pull request merged**